org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/SemanticTokensFullConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2022 Avaloq Group Ltd (http://www.avaloq.com).
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.testing;
+
+/**
+ * @author Rub�n Porras Campo - Initial Contribution and API
+ */
+public class SemanticTokensFullConfiguration extends TextDocumentConfiguration {
+	private String expectedText = "";
+
+	public String getExpectedText() {
```

Comment:
I'd appreciate if we either use a richer object to encode the "expectedText" or add a comment what is to be expected from the expected text.

Reply:
I will not be able to do that today, but if we merge this PR before the next milestone is package, I will submit a new PR and improve on it.